rs/nns/integration_tests/src/registry_get_changes_since.rs

```diff
@@ -0,0 +1,43 @@
+use ic_base_types::{PrincipalId, PrincipalIdClass};
+use ic_nns_test_utils::{
+    common::NnsInitPayloadsBuilder,
+    state_test_helpers::{
+        registry_get_changes_since, setup_nns_canisters, state_machine_builder_for_nns_tests,
+    },
+};
+use ic_registry_transport::pb::v1::{RegistryGetChangesSinceResponse, RegistryError, registry_error};
+
+#[test]
+fn test_disallow_opauqe_callers() {
```

Comment:
Is there already a test somewhere that exercises get_changes_since with a caller that is valid, and we can verify that it still works?